examples/camera-app/linux/src/clusters/chime/chime-manager.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "chime-manager.h"
+#include <iostream>
+#include <lib/support/logging/CHIPLogging.h>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::Chime;
+using namespace chip::app::Clusters::Chime::Structs;
+
+using namespace Camera;
+
+// Chime Cluster Methods
+CHIP_ERROR ChimeManager::GetChimeSoundByIndex(uint8_t chimeIndex, uint8_t & chimeID, MutableCharSpan & name)
+{
+    if (chimeIndex >= ArraySize(mChimeSounds))
+    {
+        return CHIP_ERROR_PROVIDER_LIST_EXHAUSTED;
+    }
+    chimeID = mChimeSounds[chimeIndex].chimeID;
+
+    return chip::CopyCharSpanToMutableCharSpan(mChimeSounds[chimeIndex].name, name);
```

Comment:
```suggestion
    return CopyCharSpanToMutableCharSpan(mChimeSounds[chimeIndex].name, name);
```
and similar elsewhere.